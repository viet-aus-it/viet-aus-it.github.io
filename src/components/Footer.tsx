import { FC } from 'react';

const Footer: FC = () => {
  const CurrentYear = new Date().getFullYear();

  return (
    <footer>
      {`Â© ${CurrentYear}, Built with `}
      <a href="https://www.gatsbyjs.org">Gatsby</a>
      {` Hosted on `}
      <a href="https://pages.github.com/">GitHub Pages</a>
    </footer>
  );
};

export default Footer;
