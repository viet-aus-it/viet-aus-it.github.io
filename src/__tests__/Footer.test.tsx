import { render } from '@testing-library/react';
import Footer from '../components/Template/Footer';

describe('Footer', () => {
  it('should render without crashing', () => {
    const tree = render(<Footer />);
    expect(tree).toMatchSnapshot();
  });
});
