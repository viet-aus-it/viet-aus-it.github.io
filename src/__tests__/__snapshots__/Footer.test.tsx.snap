// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render without crashing 1`] = `
<footer>
  Â© 2020, Built with 
  <a
    href="https://www.gatsbyjs.org"
  >
    Gatsby
  </a>
   Hosted on 
  <a
    href="https://pages.github.com/"
  >
    GitHub Pages
  </a>
</footer>
`;
